package.json

```diff
@@ -1,15 +1,38 @@
 {
   "private": true,
   "devDependencies": {
+    "@angular/common": "^11.2.13",
```

Comment:
Are the angular dependencies supposed to be pulled up onto the root package.json? Or are these hoisted from packages/angular?

Reply:
Right, so this is one of the discussions we should have.

There are two schools of thought on this one, and both have pros/cons:
- In mono-repo you should have a one-version policy and there fore you should have one place to manage all of your versions etc... The downside is that you will need to have a way to make package.json for individual sub-packages. This is what Angular uses.
- The other option is to have a mano-repo, but really have a different version for each sub-package. Which I think is what you are suggesting.


A generously don't know what the right answer is here. I have discussed this for a bit with @steve8708 and we decided to go the first route, but I think it is a bit riskier. But would love to know what other opinions are.

Reply:
Mono-repo does not really mean anything. It is whatever you define it to be.
Yes I went down path 1 where the versions are shared between packages but ran into an issue, so I am abandoning it and going down path 2 which is what you are advocating here.